### PR TITLE
Changelog entries for the first batch of changes in 10.0.8

### PR DESCRIPTION
**Pull request by PVince81**
See title.

Please review.

- [x] should we keep the "Bump" entries from dependabot or summarize them into a single "updated many libs" entry ? @DeepDiver1975 

We can first merge this, another train load of changes will arrive once the next big batch of PRs was merged.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,77 @@ All notable changes to this project will be documented in this file.
...
+- Guide users to also check spelling for typos in federated share id - [#30355](https://github.com/owncloud/core/issues/30355)
+- Fixed issue with number of hidden files not updating on renaming a file - [#30359](https://github.com/owncloud/core/issues/30359)
+- Fix deleted items auto expiration for users with no quota - [#30163](https://github.com/owncloud/core/issues/30163)
+- Fix validation for new encryption storage key location - [#30357](https://github.com/owncloud/core/issues/30357)
+- Fix some CSRF issues on Webdav endpoint by only checking for POST method - [#30358](https://github.com/owncloud/core/issues/30358)
+- Prevent share icon from shrinking with long texts - [#30352](https://github.com/owncloud/core/issues/30352)
+- Catch session unavailable exception - [#30347](https://github.com/owncloud/core/issues/30347)
+- Fixed regression where a user could not set own email address in the settings page - [#30319](https://github.com/owncloud/core/issues/30319)
+- Catch BadResponseException when trying to upload - [#30316](https://github.com/owncloud/core/issues/30316)
```
Comment (phil-davis) [inline CHANGELOG.md]: This is only an integration test fix.

**Review by phil-davis — APPROVED** [review]
You did a good job of filtering out the many integration/UI test-only changes. I only found 1.

File: CHANGELOG.md (hunk 1 above)
Comment (PVince81) [reply CHANGELOG.md]: Thanks, removed

**Review by pmaier1 — APPROVED** [review]
:+1: I like mentioning the bumps in detail!

**Comment by PVince81** [comment]
last changelog commit: 4656391bfe8c2a8748d0616320e015cd4d5073ae (for future reference)

**Pull request merged**